wrapper/src/main/java/software/amazon/jdbc/dialect/DialectManager.java

```diff
@@ -144,6 +144,7 @@ public Dialect getDialect(
     if (driverProtocol.contains("mysql")) {
       RdsUrlType type = this.rdsHelper.identifyRdsType(host);
       if (type.isRdsCluster()) {
+        this.canUpdate = true;
```

Comment:
Is this canUpdate valid for every RDS Cluster, or just for the new Multi AZ ones?

Reply:
The `Multi-AZ DB Cluster` is a product name with very specific supported feature: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/multi-az-db-clusters-concepts.html It's a bit unclear on whether `canUpdate` is valid for every RDS cluster because we can hardly define `every RDS cluster`.

The overall "update chain" is Aurora -> MultiAZDbCluster -> Other generic rds DBs -> generic DBs, from explicits to implicits.

Reply:
update is probably the wrong word. Should be upgrade